contentpacks/khanacademy.py

```diff
@@ -567,14 +568,56 @@ def retrieve_kalite_data(lang="en", force=False) -> list:
 
     url = url.format(projection=json.dumps(projection), lang=lang)
 
-    node_data_path = download_and_clean_kalite_data(url, lang=lang, ignorecache=force, filename="nodes.json")
+    node_data_path = download_and_clean_kalite_data(url, lang=lang, ignorecache=force, filename=filename)
 
     with open(node_data_path, 'r') as f:
         node_data = ujson.load(f)
+    
+    if not lang == "en" and not no_dubbed_videos == True:
+        node_data = addin_dubbed_video_mappings(node_data, lang) 
 
     return node_data
 
 
+def addin_dubbed_video_mappings(node_data, lang="en"):
+    # Get the dubbed videos from the spreadsheet and substitute them 
+    # for the video, and topic attributes of the returned data struct.
+
+    # Get the list of video ids from dubbed video mappings
+    lang_name = get_lang_name(lang).lower()
+    dubbed_videos_data = pkgutil.get_data('contentpacks', "resources/dubbed_video_mappings.json")
+    dubbed_videos_load = ujson.loads(dubbed_videos_data)
+    dubbed_videos_list = dubbed_videos_load[lang_name]
+
+    # Get the current youtube_ids, and topic_paths form the khan api node data.
+    youtube_ids = [node.get("youtube_id") for node in node_data if node.get("kind") == "Video"]
```

Comment:
Instead of using the "Video" and "Topic" strings directly, please use the enums defined in `ContentType`: https://github.com/fle-internal/content-pack-maker/blob/master/contentpacks/utils.py#L26-L29